pkg/appregistry/appregistry_test.go

```diff
@@ -0,0 +1,22 @@
+package appregistry_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/operator-framework/operator-marketplace/pkg/appregistry"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRetrieveAll(t *testing.T) {
+	factory := appregistry.NewClientFactory()
+
+	client, err := factory.New("appregistry", "http://localhost:5000/cnr")
```

Comment:
Not sure, if the assumption is we have some service running on 5000? Or did you mock it somewhere?

Reply:
This test is hanging around here for debugging purposes. It will be replaced by a unit test.